db/migrate/003_add_collapsed_ids_to_issues.rb

```diff
@@ -0,0 +1,5 @@
+class AddCollapsedIdsToIssues < ActiveRecord::Migration[6.0]
+  def change
+    add_column :issues, :collapsed_ids, :string, limit: 1000

```

Comment:
Where did you find out that `:text` is not supported on oracle?

According to what it is written here https://api.rubyonrails.org/v7.1.3.4/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_column should be generally balable. It is true that I didn't find code for Oracle on rails sources

Reply:
string e tradus in varchar si e mai universal, e mult mai safe